.github/workflows/test.yaml

```
@@ -0,0 +1,27 @@
+# This is a GitHub workflow defining a set of jobs with a set of steps.
+# ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+#
+name: Test
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  test-black-autoformatting:
```

Vi kan strunta i att inkludera denna detalj - eller vad tycker du?

```suggestion
  test-autoformatting:
```